src/main/webapp/app/exercises/modeling/manage/modeling-exercise.service.ts

```diff
@@ -94,6 +95,12 @@ export class ModelingExerciseService {
             .pipe(map((response: HttpResponse<ModelingPlagiarismResult>) => response.body!));
     }
 
+    convertToPdf(model: string): Observable<any> {
+        return this.http
+            .post(`${SERVER_API_URL}api/apollon-convert/pdf`, { model }, { observe: 'response', responseType: 'blob' })
+            .pipe(map((response: HttpResponse<Blob>) => downloadStream(response.body, 'application/pdf')));
```

Comment:
A friendly file name like `exercise123-sample-solution.pdf` would be nice.

Reply:
exercise123-example-solution.pdf is the name now for the pdf.